<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOControl</class>
 <widget class="QDialog" name="IOControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IO Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>IO</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbIn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>IN</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbOut">
        <property name="text">
         <string>OUT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Refresh Input</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Strobe</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="cbEnableStrobe">
        <property name="text">
         <string>Enable strobe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbStrobePolarity">
        <property name="text">
         <string>Invert strobe polarity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Trigger</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cbTriggerMode">
        <property name="text">
         <string>Enter trigger mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbTriggerPolarity">
        <property name="text">
         <string>Invert trigger polarity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
